netcode.py

```diff
@@ -112,6 +112,16 @@ def receive_any(self) -> Message:
 
         return self.receive_matching(lambda _message: True)
 
+    def flush_mailbox(self) -> list[Message]:
+        """
+        Remove all messages from the mailbox and return them.
+        """
+
+        # TODO: This possibly isn't thread safe?
```

Comment:
Nie powinno być problemu raczej, o ile python może być wielowątkowy, to w jednym momencie tylko jeden wątek będzie działał i powinien być zawsze "thread safe", a przynajmniej tak twierdzi ten pan https://youtu.be/AZnGRKFUU0c?t=31

Reply:
Fakt, nie wziąłem pod uwagę, że trzeba całość na raz wykonać, mb